ExampleMod/Content/Items/Placeable/Furniture/ExampleDresser.cs

```diff
@@ -0,0 +1,31 @@
+using Terraria;
+using Terraria.ID;
+using Terraria.ModLoader;
+
+namespace ExampleMod.Content.Items.Placeable.Furniture
+{
+	public class ExampleDresser : ModItem
+	{
+		public override void SetStaticDefaults() {
+			Tooltip.SetDefault("This is a modded dresser.");
+
+			Item.ResearchUnlockCount = 1;
```

Comment:
Would be nice to have this be the default in a separate PR as you mentioned before. Then a lot of content won't even need a `SetStaticDefaults`